graphql/journey.test.js

```diff
@@ -151,6 +151,58 @@ describe("the graphql journey", () => {
       .catch((e) => fail("it should not have error'd" + e));
   });
 
+  test("graphql get bm25 with query (without properties)", () => {
```

Comment:
graphql sent to server does not contain bm25 string:
`{Get{Article{_additional { id }}}}`